lib/index.html

```diff
@@ -97,9 +97,13 @@
               <p class="soft">Safely ejected and ready for use</p>
 
               <div class="space-vertical-large">
-                <hero-button ng-click="app.restart()">
+                <hero-button ng-click="app.restart(app.selection.getImage())">
```

Comment:
This looks nice, however the meaning of passing an image to a restart function is not very straightforward at first sight. A more expressive approach would be allowing to pass an opt-in option to `app.restart()` asking to preserve the image, so we would do something like:

```html
<hero-button ng-click="app.restart({ preserveImage: true })">
```

Which reads a lot better and natural.

The `app.restart()` options could be directly passed to `app.selection.clear()`, where you can implement and unit test.